docs/token_provisioning.md

```diff
@@ -0,0 +1,107 @@
+# Fobnail Token provisioning
+
+The process of Token provisioning is initialized by [Platform Owner
+application](/building/#platform-owner).
+
+## Prerequisites
+
+Platform Owner binary expects two files: PO certificate chain `cert_chain.pem`
+and private key `po_priv_key.pem`, in that order. For quick start both of those
+files can be produced using [these TL;DR
+instructions](/keys_and_certificates/#tldr-version).
+
+Certificate chain must begin with the root CA that was passed as
+`FOBNAIL_PO_ROOT` when [Token firmware was
+built](/building/#environment-variables-common-for-both-targets).
+
+## Provisioning instructions
+
+Instructions below assume that `fobnail-platform-owner` was installed to a
+directory in `$PATH` variable and both required PEM files are in current
+directory. The process differs slightly between physical Token and emulated one,
+choose proper version before continuing.
+
+=== "Physical Fobnail Token"
```

Comment:
@pietrushnic please let me know what do you think about presenting different paths in this way. If this is clear enough I can change building instructions to use tabs, too.

Reply:
Yes, this is a very clear and way better solution.

Reply:
Done in https://github.com/fobnail/docs/pull/47/commits/26e40f6570109a81d7d960aa4e629f433e3f216c